test/cql-pytest/test_compactionstats.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pytest
+
+import nodetool
+from util import new_test_table
+
+
+def get_compaction_stats(cql, table):
+    ks, cf = table.split('.')
+    stats = nodetool.compactionstats(cql)
+    tasks = 0
+    for s in stats:
+        if s['ks'] == ks and s['cf'] == cf:
+            tasks += int(s['task'])
+    return tasks, stats
+
+@pytest.mark.parametrize("compaction_strategy", ["LeveledCompactionStrategy", "SizeTieredCompactionStrategy", "TimeWindowCompactionStrategy"])
+def test_compactionstats_after_major_compaction(scylla_only, cql, test_keyspace, compaction_strategy):
+    """
+    Test that compactionstats show no pending compaction after major compaction
+    """
+    num_sstables = 16
+    extra_strategy_options = ""
+    if compaction_strategy == "LeveledCompactionStrategy":
+        extra_strategy_options = ", 'sstable_size_in_mb':1"
+        num_sstables *= 4   # Need enough data to trigger level 0 compaction
```

Comment:
Wow, this test needs to do over a hundred flushes? 128 KB in each file? :-( How much time does it take to run? :-(
I don't understand why you can't test it with two sstables with 10 bytes in each... You can run a major compaction even on two sstables with 10 bytes each, no?

Reply:
This is needed for verifying that leveled compaction strategy estimates >0 pending conpactions before major compaction.
In level0 it needs the total size of the sstables to be greater than 4 times max_sstable_size_mb.

See https://github.com/scylladb/scylladb/blob/5a470b2bfbe6fdb371bbde2a2f8987d6618dd265/compaction/leveled_manifest.hh#L497